scripts/downloader.py

```
@@ -21,35 +21,32 @@
 import os
 import time
 from PIL import Image
-
+from tqdm import tqdm
 from constants import ROOT_DIR, SEREBII_BASE_URL, SESSION
 from utils import serebii_slug
 
 def download_images(cards, prefix):
     webp_dir, png_dir = os.path.join(ROOT_DIR, "images", "webp", "cards", prefix), os.path.join(ROOT_DIR, "images", "png", "cards", prefix)
     os.makedirs(webp_dir, exist_ok=True); os.makedirs(png_dir, exist_ok=True)
```

Ruff (0.16.1)

[error] Multiple statements on one line (semicolon)

(E702)